lib/request.ts

```diff
@@ -51,7 +51,7 @@ export default class CIORequest {
     const headers = {
       Authorization: this.auth,
       'Content-Type': 'application/json',
-      'Content-Length': body ? body.length : 0,
+      'Content-Length': body ? Buffer.from(body).length : 0,
```

Comment:
Rather than convert the body into a buffer without using it, let's instead use the [`Buffer.byteLength`](https://nodejs.org/dist/latest-v14.x/docs/api/buffer.html#buffer_static_method_buffer_bytelength_string_encoding) static method to check the length. At the same time, the encoding should be explicitly set to `utf8` just in case (that's the default)

Reply:
Yes good idea!